ui/app/components/app-item/deployment.hbs

```diff
@@ -7,29 +7,66 @@
   <hr>
   <LinkTo
     data-test-deployment-list-item
-    title={{ concat
-      "v" @deployment.sequence ": "
-      (t (concat "page.deployments.status_prefix." @deployment.status.state))
-      (component-name @deployment.component.name)}}
+    title={{concat (t "page.deployment.title") " v" @deployment.sequence }}
     @route="workspace.projects.project.app.deployment.deployment-seq"
     @models={{array @deployment.sequence}}
   >
     <b class="badge badge--version">v{{@deployment.sequence}}</b>
-    <OperationStatusIndicator::Deployment @operation={{@deployment}}>
-      <:small-text>
-        {{#if (not-eq @latest.id @deployment.id)}}
-        <span class="replacement-info">
-          {{t "page.deployments.replaced_label"}}<b class="badge badge--version">v{{@latest.sequence}}</b>
+    <div class="app-item__deployment-state">
+      {{!-- ROW: status report + deployment url --}}
+      {{#if @deployment.statusReport.health}}
+        {{#let (hash
+          deploymentHealth=@deployment.statusReport.health.healthStatus
+          urlOrStatus=(or
+            (clean-up-url @deployment.preload.deployUrl)
+            (t (concat "status_report_indicator.label."
+              (downcase @deployment.statusReport.health.healthStatus))
+            )
+          )
+        ) as |vars|}}
+          <span
+            data-test-health-status={{downcase vars.deploymentHealth}}
+            class="deploy-url__status deploy-url__status--{{downcase vars.deploymentHealth}}"
+          />
+          <span class="deploy-url" title={{vars.urlOrStatus}}>
+            {{vars.urlOrStatus}}
+          </span>
+        {{/let}}
+      {{/if}}
+      {{!-- ROW: operation status --}}
+      {{#let @deployment.status as |status|}}
+        <FlightIcon
+          data-test-time-icon
+          @name={{or
+            (and (eq status.state 0) "help")
+            (and (eq status.state 1) "clock")
+            (and (eq status.state 2) "check")
+            (and (eq status.state 3) "alert-triangle")
+          }}
+          @size="16"
+        />
+        <span data-test-operation-status>
```

Comment:
Are we intending to add the tooltips back to these at some point? Or are we reserving that information for the detail page?

Reply:
I assumed it'd be too busy with the tooltips, but @almonk can weigh in as well

Reply:
I’m fine with that. Probably also better from an a11y perspective, so we’re not nesting semi-interactive stuff within a link.